CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+2.1.0
```

Comment:
Please remove any changes to the package version. That only gets decided at release time, when all the changes in the release need to be examined together to determine the next version.